### ISSUE TITLE
Compatibility with pyproj 2

### ISSUE THREAD

**Issue by sscherfke** [matplotlib/basemap#458]
basemap is not yet compatible with pyproj 2.* (which uses proj.4 v6):
```
...
    from mpl_toolkits.basemap import Basemap, Polygon
.../lib/python3.7/site-packages/mpl_toolkits/basemap/__init__.py:152: in <module>
    epsgf = open(os.path.join(pyproj.pyproj_datadir,'epsg'))
E   AttributeError: module 'pyproj' has no attribute 'pyproj_datadir'
```

**Comment by huisaddison** [comment]
Yes, it looks like some people are working on a fix at https://github.com/matplotlib/basemap/pull/454

I was able to continue using basemap by reverting pyproj to version 1.9.6

**Comment by WeatherGod** [comment]
closed by #454
